README.md

```diff
@@ -12,5 +12,17 @@ This is a demo of the various [Meteoric](https://github.com/meteoric) packages:
 
 See it in action here: [meteoric.github.io](http://meteoric.github.io)
 
+## Running locally
+
+Due to [some](https://github.com/meteoric/demo/issues/4) [issues](https://github.com/meteoric/demo/issues/6), you need to run the following for a fully-functional demo:
```

Comment:
Maybe we should link to the [README for `ionic-sass`](https://github.com/meteoric/ionic-sass#installation) since that's where the issue is actually happening?